src/command/Menus.js

```diff
@@ -1067,7 +1067,11 @@ define(function (require, exports, module) {
         }
 
         // Add a file-menu class to hide non-file items from file context menu
-        $("#project-context-menu").toggleClass("file-menu", mouseOrLocation.fileMenu);
+        if(mouseOrLocation.fileMenu) {
```

Comment:
Yup I think it needs to be boolean. 
Since 
```js
$("#project-context-menu").toggleClass("file-menu") 
```
simply toggles the value, and 
```js
$("#project-context-menu").toggleClass("file-menu", parameter)
```
sets or unsets it on the basis of the parameter.
But in this case the parameter being passed is undefined, so it simply toggles when it actually should take the second parameter into consideration.

```js
$("#project-context-menu").toggleClass("file-menu", !!(mouseOrLocation.fileMenu))
``` 
seems interesting. First time I saw this.